synthetics_POC/terraform/README.md

```diff
@@ -1,7 +1,7 @@
 ## Create secret from existing ksynth installation
 
-To create a secret that can be used in terraform automation script for new instances you have to follow this steps:
-- ssh in to VM which has ksynth agent installed
+To create a secret that can be used in terraform automation script for new instances you have to follow these steps:
+- ssh into VM which has ksynth agent installed
```

Comment:
I do understand that you've just fixed typos and grammar in the sentence, but the instruction below are bad and will cause `ksynth` instances deployed with the replicated `ksynth.id` to not work. The only use case for this procedure is to allow in-place update/upgrade of a VM running `ksynth` (note that the `ksynth` app upgrades itself by default without any need to follow a specific procedure). Also installing AWS CLI in a VM running `ksynth`, as opposed to copying the `ksynth.id` from the VM to machine that already has the CLI installed, is IMHO a bad idea.

Please, change the title and text to:

```
## Preserving ksynth instance identity across VM rebuild

The `ksynth.id` file contains unique identity for a `ksynth` instance. It needs to be preserved across VM upgrade in order to keep the instance associated with the tracking data in Kentik.

In case of `ksynth` hosted in AWS VM, this can be achieved by storing the `ksynth.id` in AWS Secrets Manager, using following steps:
- install AWS CLI on your workstation [install aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and configure it for the target account (`aws configure`)
- copy `ksynth.id` from the running VM to your local workstation: `scp <th_VM>:/var/lib/ksynth/ksynth.id <ksynth_instance_name>.id`
- create entry in AWS Secrets Manager: `aws secretsmanager create-secret --name <ksynth_instance_name> --secret-binary fileb:<ksynth_instance_name>.id`

Note that you have to use unique `ksynth_instance_name` for each unique `ksynth` instance (i.e. for each instance represented as a separate agent in Kentik. The `ksynth.id` identity cannot be shared among instances.
```

It is actually possible that the whole `synthetics_POC` will need to be revised or removed.

Reply:
I'm not familiar with the ksynth related procedure yet, only corrected some typos in the text.
Updating the text to suggestion.